<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MTPageNym_Source</class>
 <widget class="QWizardPage" name="MTPageNym_Source">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Source Label:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditSource"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="getAddress">
     <property name="text">
      <string>Get Address</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="labelExplanation">
     <property name="text">
      <string>Explanation Label</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="labelExtra">
     <property name="text">
      <string>Extra Info Label</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
